quinn-proto/src/connection/mod.rs

```diff
@@ -70,8 +70,18 @@ where
     handshake_cid: ConnectionId,
     /// The CID the peer initially chose, for use during the handshake
     rem_handshake_cid: ConnectionId,
+    /// Cid rotation
```

Comment:
If this comment is intended to describe the entire block of variables that follows, it should not be a doc comment (which will get attached specifically to the next field instead), should be preceded by an empty line, and should be bracketed by empty comments.

Reply:
Ack